README.md

```
@@ -1,220 +1,390 @@
+    end
+    
+    Celery->>Orchestrator: All Results
+    Orchestrator->>Orchestrator: Generate Scorecard
+    Orchestrator->>API: Complete Report
+    API->>Frontend: Report Ready
+    Frontend->>User: Display Results + PDF
+```
+
+### Directory Structure
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)